File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opencv/callopencv.java
package org.firstinspires.ftc.teamcode.opencv;

import com.acmerobotics.dashboard.FtcDashboard;
import com.qualcomm.robotcore.eventloop.opmode.Disabled;
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
import com.qualcomm.robotcore.eventloop.opmode.TeleOp;

import org.firstinspires.ftc.teamcode.robot.WoENrobot;

@TeleOp
//@Disabled
public class callopencv extends LinearOpMode {

    @Override
    public void runOpMode() {
        new WoENrobot();
       // WoENrobot.forceInitRobot(this);
        WoENrobot.openCVNode.initialize(this);
        waitForStart();
        FtcDashboard.getInstance().startCameraStream(WoENrobot.openCVNode.getWebcam(),0);
        while (opModeIsActive()) {
            telemetry.addData("mean", WoENrobot.openCVNode.getMean());
            telemetry.addData("getAspectRatio", WoENrobot.openCVNode.getAspectRatio());
            telemetry.addData("getStackSize", WoENrobot.openCVNode.getStackSize());
            telemetry.update();
        }
        FtcDashboard.getInstance().stopCameraStream();
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/test/callopencv.kt
package org.firstinspires.ftc.teamcode.test

import com.qualcomm.robotcore.eventloop.opmode.TeleOp
import com.qualcomm.robotcore.eventloop.opmode.Disabled
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode
import org.firstinspires.ftc.teamcode.robot.WoENrobot
import com.acmerobotics.dashboard.FtcDashboard

@TeleOp
@Disabled
class callopencv : LinearOpMode() {
    override fun runOpMode() {
        // WoENrobot.forceInitRobot(this);
        WoENrobot.openCVNode.initialize(this)
        waitForStart()
        FtcDashboard.getInstance().startCameraStream(WoENrobot.openCVNode.webcam, 0.0)
        while (opModeIsActive()) {
            telemetry.addData("mean", WoENrobot.openCVNode.mean)
            telemetry.addData("getAspectRatio", WoENrobot.openCVNode.aspectRatio)
            telemetry.addData("getStackSize", WoENrobot.openCVNode.stackSize)
            telemetry.update()
        }
        FtcDashboard.getInstance().stopCameraStream()
    }
}